### PR TITLE
update user guide for sending alert to notification manager

### PR DESCRIPTION
**Pull request by wanjunlei**
Signed-off-by: wanjunlei <wanjunlei@yunify.com>

### CODE REVIEW

File: content/en/monitoring/notification-manager.md [hunk 1]
```diff
@@ -66,13 +66,15 @@ Send auditing alerts to Notification Manager:
         "group_interval": "30s"
 ```
 
+> These configurations are the default configuration. If you do not want to receive a certain type of alert, you can delete the corresponding configuration.
```
Comment (FeynmanZhou) [inline content/en/monitoring/notification-manager.md:69]: The above is the default configuration.

Comment (wanjunlei) [reply content/en/monitoring/notification-manager.md:69]: yes

File: content/en/monitoring/notification-manager.md [hunk 2]
```diff
@@ -388,7 +390,7 @@ spec:
 EOF
 ```
 
-Slack receiver labeled with `type: tenant` only receive notification happend in the namespace which user be in. If you want to receive notification without namespace, you can create a global slack receiver labeled with `type: global` like this.
+Slack receiver labeled with `type: tenant` only receive notification happened in the namespace which user be in. If you want to receive notification without namespace, you can create a global slack receiver labeled with `type: global` like this.
```
Comment (FeynmanZhou) [inline content/en/monitoring/notification-manager.md:393]: Slack receiver labeled with `type: tenant` only receive notification happened in the namespace which user be in => Slack receiver labeled with `type: tenant` only receive notification happened in the namespace where the users are  invited

Is this description more exact?

Comment (FeynmanZhou) [reply content/en/monitoring/notification-manager.md:393]: @Sherlock113 Please help me to verify if my comment much is more exact here.

Comment (Sherlock113) [reply content/en/monitoring/notification-manager.md:393]: To my understanding,

1. "If you want to receive notification without namespace": this is ambiguous to me. My guess is that you mean you want users to receive notifications no matter what namespace they are in.

2. "slack receiver" is not a person here. Rather, it receives notifications and send them to the actual user (person).

Based on the above, here is my version:

The slack receiver labeled with `type: tenant` only receives notifications that happen in the namespace where the tenant has already existed. If you want them to receive notifications regardless of the namespace, you can create a global slack receiver labeled with `type: global` as follows:

Comment (FeynmanZhou) [reply content/en/monitoring/notification-manager.md:393]: @wanjunlei Please take a look at @Sherlock113 's comment, I agree with this one.

File: content/en/monitoring/notification-manager.md (hunk 1 above)
Comment (FeynmanZhou) [reply content/en/monitoring/notification-manager.md:69]: `These configurations are the default configuration` => `The above is the default configuration`

File: content/en/monitoring/notification-manager.md [hunk 3]
```diff
@@ -20,8 +20,8 @@ Supported receivers includes:
 
 ### Config Prometheus Alertmanager to send alerts to Notification Manager
 
-Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubsphere.
-To receive Alertmanager alerts, eidt the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config like below to the `receivers` section of alertmanager.yaml:
+Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubesphere.
+To receive Alertmanager alerts, edit the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config and route like below to the alertmanager.yaml:
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:24]: To receive Alertmanager alerts, edit the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config and route like below to the alertmanager.yaml:
=>
To receive Alertmanager alerts, KubeSphere already added Alertmanager webhook and route configurations like below ( by editing the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system ):

File: content/en/monitoring/notification-manager.md [hunk 4]
```diff
@@ -20,8 +20,8 @@ Supported receivers includes:
 
 ### Config Prometheus Alertmanager to send alerts to Notification Manager
 
-Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubsphere.
-To receive Alertmanager alerts, eidt the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config like below to the `receivers` section of alertmanager.yaml:
+Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubesphere.
+To receive Alertmanager alerts, KubeSphere already added Alertmanager webhook and route configurations like below ( by editing the Secret alertmanager-main in the namespace `kubesphere-monitoring-system ):
```
Comment (FeynmanZhou) [inline content/en/monitoring/notification-manager.md:24]: I think there is a Single quote missing after `kubesphere-monitoring-system`.

File: content/en/monitoring/notification-manager.md [hunk 5]
```diff
@@ -129,7 +131,7 @@ spec:
 EOF
 ```
 
-The emailConfigSelector is a selector to selecte EmailConfig for email receiver, if the emailConfigSelector is nil, receiver will use the default email config. You can create a default email config like this.
+The emailConfigSelector is a selector to select EmailConfig for email receiver, if the emailConfigSelector is nil, receiver will use the default email config. You can create a default email config like this.
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:134]: if the emailConfigSelector is nil => if the emailConfigSelector is not set

File: content/en/monitoring/notification-manager.md [hunk 6]
```diff
@@ -245,9 +247,9 @@ spec:
 EOF
 ```
 
->WechatApiCorpId is the id of your Wechat Work. WechatApiAgentId is the id of app which sending message to user in your Wechat Work, wechatApiSecret is the secret of this app, you can get these two parameters in App Managerment of your Wechat Work. Note that any user, party or tag who wants to rerceive notifications must be in the allowed users list of this app.
+> WechatApiCorpId is the id of your Wechat Work. WechatApiAgentId is the id of app which sending message to user in your Wechat Work, wechatApiSecret is the secret of this app, you can get these two parameters in App Management of your Wechat Work. Note that any user, party or tag who wants to receive notifications must be in the allowed users list of this app.
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:250]: WechatApiAgentId is the id of app which sending message to user in your Wechat Work =>
WechatApiAgentId is the id of the app sending message to user in your Wechat Work

File: content/en/monitoring/notification-manager.md [hunk 7]
```diff
@@ -245,9 +247,9 @@ spec:
...
 
-The wechatConfigSelector is a selector to selecte WechatConfig for wechat receiver, if the wechatConfigSelector is nil, wechat receiver will use the default wechat config. You can create a default wechat config like this.
+The wechatConfigSelector is a selector to select WechatConfig for wechat receiver, if the wechatConfigSelector is nil, wechat receiver will use the default wechat config. You can create a default wechat config like this.
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:252]: if the wechatConfigSelector is nil =>
if the wechatConfigSelector is not set

File: content/en/monitoring/notification-manager.md [hunk 8]
```diff
@@ -355,9 +357,9 @@ spec:
 EOF
 ```
 
->Slack token is the OAuth Access Token or Bot User OAuth Access Token when you create a slack app. This app must have the scope chat:write. The user who created the app or bot user must be in the channel which you want to send notification to.
+> Slack token is the OAuth Access Token or Bot User OAuth Access Token when you create a slack app. This app must have the scope chat:write. The user who created the app or bot user must be in the channel which you want to send notification to.
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:360]: The user who created the app or bot user must be in the channel which you want to send notification to. =>
The user who creates the app or bot user must be in the channel to which you want to send notification.

File: content/en/monitoring/notification-manager.md [hunk 9]
```diff
@@ -355,9 +357,9 @@ spec:
...
 
-The slackConfigSelector is a selector to selecte SlackConfig for slack receiver, if the slackConfigSelector is nil, slack receiver will use the default slack config. You can create a default slack config like this.
+The slackConfigSelector is a selector to select SlackConfig for slack receiver, if the slackConfigSelector is nil, slack receiver will use the default slack config. You can create a default slack config like this.
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:362]: if the slackConfigSelector is nil =>
if the slackConfigSelector is not set

File: content/en/monitoring/notification-manager.md [hunk 10]
```diff
@@ -405,4 +407,4 @@ spec:
 EOF
 ```
 
->Global slack receiver use the default slack config.
+> Global slack receiver use the default slack config.
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:410]: Global slack receiver use the default slack config. =>
Global slack receiver will use the default slack config.

File: content/en/monitoring/notification-manager.md [hunk 11]
```diff
@@ -390,7 +390,7 @@ spec:
 EOF
 ```
 
-Slack receiver labeled with `type: tenant` only receive notification happened in the namespace which user be in. If you want to receive notification without namespace, you can create a global slack receiver labeled with `type: global` like this.
+The slack receiver labeled with `type: tenant` only receives notifications that happen in the namespace where the tenant has already existed. If you want them to receive notifications regardless of the namespace, you can create a global slack receiver labeled with type: global as follows:
```
Comment (benjaminhuo) [inline content/en/monitoring/notification-manager.md:393]: =>
Slack receivers with label `type: tenant` can only receive notifications from the namespace to which the specified tenant user has access. If you want them to receive notifications from all namespaces or even without a namespace label, you can create a global slack receiver with label `type: global` as below:

**Comment by benjaminhuo** [comment]
@wanjunlei , pls see my comments above

**Pull request merged**